vdf/src/proof_of_time.rs

```diff
@@ -102,18 +109,27 @@ where
     let powers = iterate_squarings(x.clone(), powers_to_calculate.iter().cloned());
     let y = &powers[&iterations];
     let identity = &x.identity();
-    let proof = super::proof_pietrzak::generate_proof(x, iterations, delta, y.clone(), &powers, identity, &generate_r_value, int_size_bits)?;
+    let proof = super::proof_pietrzak::generate_proof(
+        x,
+        iterations,
+        delta,
+        y.clone(),
+        &powers,
+        identity,
+        &generate_r_value,
+        int_size_bits,
+    )?;
     let proof_len = proof.len();
     let element_length = 2 * ((int_size_bits + 16) >> 4);
     let proof_len_in_bytes = (proof_len + 1) * element_length;
     let mut v = Vec::with_capacity(proof_len_in_bytes);
-    for _ in 0..proof_len_in_bytes {
-        v.push(0)
-    }
-    y.serialize(&mut v[0..element_length])?;
+    v.resize(proof_len_in_bytes, 0);
```

Comment:
Would `let mut v = vec![0; proof_len_in_bytes]` do the same?

Reply:
It would, thank you!

Reply:
Done.